corehq/apps/export/forms.py

```diff
@@ -121,10 +121,13 @@ def __init__(self, has_form_export_permissions, has_case_export_permissions, *ar
                     ),
                     data_bind="visible: showAppType()",
                 ),
-                crispy.Field(
-                    'application',
-                    placeholder=_("Select Application"),
-                    data_bind="value: application",
+                crispy.Div(  # Case export fields
+                    crispy.Field(
+                        'application',
+                        placeholder=_("Select Application"),
+                        data_bind="value: application",
+                    ),
+                    data_bind="visible: isFormModel()",
```

Comment:
You could also move this inside the "form export fields" div below, since it's now only applicable there.

Reply:
https://github.com/dimagi/commcare-hq/pull/33088/commits/64cd7d452379ab0c578815d0d6dc7dd377ba08ad